app/js/components/manageApps/ManageApps.jsx

```diff
@@ -119,8 +119,12 @@ export default class ManageApps extends React.Component {
       this.handleApplist();
     }).catch(
       (error) => {
-        location.href = `${location.href.substr(0, location.href.indexOf(location.href.split('/')[4]))}login.htm`;
-        toastr.error(error);
+        error.response.status === 401 ?
```

Comment:
Are the changes in this file related to the ticket?

Reply:
Yes, this gives a better explanation in the toast that appears when one clicks start all modules and some fail to start. Originally it would inform the user the `an error of status code 500 occurred`. I thought this was uninformative and for a non technical user they would not know how to decipher the issue.